src/components/DuplicatesPage/DuplicatesPage.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import DuplicatesList from "./DuplicatesList";
+
+class DuplicatesPage extends React.Component {
+
+    componentWillMount() {
+        this.props.actions.fetchDuplicates();
+    }
```

Comment:
https://github.com/tessalt/ds9-episodeguide/issues/2  ---> use componentDidMount for data fetch. Is this layer necessary? Fetch might be performed in DuplicatesList too.